CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# processing
+
+This repositary includes python modules for running the STEMMUS-SCOPE model in a
+notebook. 
+
+The workflow is executed using python and MATLAB on a Unix-like system.
+The python packages are listed in the
+[`environment.yml`](https://github.com/EcoExtreML/processing/blob/main/environment.yml)
+file. Follow the instructions below to create conda environment and install
+MATLAB Runtime.
+
+<details>
+  <summary>Create conda environment </summary>
+
+Run the commands below in a terminal:
+
+```sh
+# Download and install Conda
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-pypy3-Linux-x86_64.sh
+bash Mambaforge-pypy3-Linux-x86_64.sh 
+-b -p ~/mamba
+
+# Update base environment
+. ~/mamba/bin/activate
+mamba update --name base mamba
+
+# Clone this repository
+git clone https://github.com/EcoExtreML/processing.git
+
+# Create a conda environment called 'stemmus' with all required dependencies
+cd processing
+mamba env create
+
+# The environment can be activated with
+. ~/mamba/bin/activate stemmus
+
+```
+</details>
+
+<details>
+  <summary>Use MATLAB </summary>
+
+To run the STEMMUS_SCOPE, you need MATLAB version `>=2019`.
+
+**On Snellius:**
+
+[Snellius](https://servicedesk.surfsara.nl/wiki/display/WIKI/Snellius) is the
+Dutch National supercomputer hosted at SURF. MATLAB Runtime is installed on
+Snellius, see the script
+[`run_jupyter_lab_snellius_dev.sh`](https://github.com/EcoExtreML/processing/blob/main/run_jupyter_lab_snellius_dev.sh)
+on how to load the module.
+</details>
+
+# Run jupyter notebook
+
+**On Snellius:**
+
+Use the script
+[`run_jupyter_lab_snellius_dev.sh`](https://github.com/EcoExtreML/processing/blob/main/run_jupyter_lab_snellius_dev.sh)
+to create a jupyter lab server on Snellius for running the notebook
+interactively.
+
+**On CRIB:**
```

Comment:
This part falls under the section "Run jupyter notebook". But it seems on CRIB the steps listed here intend to tell the user how to run python script inside matlab, rather than running jupyter notebook. It is better to create a separate section for it, with the name e.g. "Use python modules on CRIB in matlab".

(Following these steps, on CRIB it asks me to login with a matlab account. But I guess this implies that you need to have an account with valid license. So I didn't test it.)

Reply:
It is fixed.